eq-author/src/App/page/Logic/BinaryExpressionEditor/NumberAnswerSelector.js

```diff
@@ -11,6 +11,7 @@ import { rightSideErrors } from "constants/validationMessages";
 import ValidationError from "components/ValidationError";
 
 const conditions = {
+  SELECT: "Select",
```

Comment:
```suggestion
  SELECT: "",
```

Just a  wee suggestion - if we make the value saved when "Select an operator" is chosen to be an empty string the AJV validation will be more straightforward since we can use the existing keyword we have which checks for populated strings (and an empty string is more readily understood as not being a valid choice). Otherwise looks good, I'll try out in Author in a sec.